src/coreclr/gc/gc.cpp

```diff
@@ -2989,17 +2989,25 @@ gc_heap::dt_high_frag_p (gc_tuning_point tp,
             }
             else
             {
-#ifndef MULTIPLE_HEAPS
                 if (gen_number == max_generation)
                 {
                     float frag_ratio = (float)(dd_fragmentation (dynamic_data_of (max_generation))) / (float)generation_size (max_generation);
-                    if (frag_ratio > 0.65)
+                    int conserve_mem_setting = (int)GCConfig::GetGCConserveMem();
+                    if (conserve_mem_setting < 0)
+                        conserve_mem_setting = 0;
+                    if (conserve_mem_setting > 9)
+                        conserve_mem_setting = 9;
+                    float frag_limit = 1.0f - conserve_mem_setting / 10.0f;
```

Comment:
assume this is a customer facing setting? Would it be easier to document if it was represented as straight percent, instead of between 0-9?

Reply:
we specifically did not want this to be a particular internal metric in GC and instead wanted a dial so GC can change the implementation as needed.

Reply:
I consider the implementation as an internal detail. 

The intuitive meaning is simply: the higher you set it, the harder GC will try to conserve memory, at the expense of using more time in GC pauses. Plus, I didn't want this setting to imply any kind of precision - people shouldn't be tempted to complain if they set it to 54% and we use 57%.